includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -72,6 +78,42 @@ public function filter_woocommerce_admin_features( $features ) {
 		return $features;
 	}
 
+	/**
+	 * Add is-woocommerce-home body class.
+	 *
+	 * @since   1.9.5
+	 *
+	 * @param string $classes Body classes.
+	 * @return string
+	 */
+	public function filter_woocommerce_body_classes( $classes ) {
+
+		$page = PageController::get_instance()->get_current_page();
```

Comment:
Hmm, I was wondering how this fix would behave when the `woocommerce_admin_disabled` filter is ON. 

I would suggest being more defensive here and checking if the `\Automattic\WooCommerce\Admin\PageController` class exists first.

Reply:
Does this filter prevent the PageController from being loaded? Not sure, actually.

Reply:
Just tested it on a WoA site. Everything seems OK!